articles/introduction-to-lxd-container-manager/index.md

```diff
@@ -0,0 +1,38 @@
+### Introduction
+LXD is a highly effective container manager. It provides an individual experience same as that offered by virtual machines. LXD systems are unique because they operate with Linux containers. This technology was introduced by Canonical Ltd alongside other contributors. LXD is supported by numerous Linux distributions. One of the significant pillars of LXD is a powerful REST API. 
+
+### Goal
+This article discusses the key features of LXD, the benefits, as well as its releases. 
+
+### Understanding what LXD entails
+LXD is a lightweight container manager which operates in the same way as a virtual machine. It does not require a Linux kernel and CPU hardware virtualization. LXD containers are more advantageous as compared to virtual machines. This is because the server can support more processes than virtual containers. LXD offers a hypervisor that helps to manage the life cycle of containers effectively. Furthermore, LXD containers are easy to control since all operations are done via a REST API. This means that accessing a remote server or the local host is seamless.
```

Comment:
Explain what is `Linux kernel`, `Hardware virtualization`, and `hypervisor`.

Brief a little about REST APIs.